CoreML in ARKit/ViewController.swift

```diff
@@ -191,7 +187,7 @@ class ViewController: UIViewController, ARSCNViewDelegate {
         // Get Classifications
         let classifications = observations[0...1] // top 2 results
             .flatMap({ $0 as? VNClassificationObservation })
-            .map({ "\($0.identifier) \(String(format:"- %.2f", $0.confidence))" })
+            .map({ "\($0.identifier) \(String(format:"- %.2f", ($0.confidence)*100)) \("%")" })
```

Comment:
Earlier it showed only the confidence and the output used to be eg: 0.7632
Now it will show in percentage and the output comes as eg: 76.32 %